docs/source/routing/observability/debugging-client-requests.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Debugging Client Requests to GraphOS Router
+subtitle: Options for analyzing incoming requests to help debug problematic requests
+description: Learn how to use GraphOS router telemetry and GraphOS Insights to inspect and debug incoming HTTP client requests.
+---
+
+By default, the GraphOS Router doesn't generate logs for incoming HTTP requests nor export other telemetry metrics. The high request volume your router likely processes can generate significant telemetry data, increasing observability costs. For this reason, you can [configure](/graphos/routing/observability/telemetry) exactly the telemetry you want your router to collect and export.
+
+## Using GraphOS Insights
+
+Without enabling additional telemetry, the easiest way to analyze GraphQL operation data is through [GraphOS Insights](/graphos/platform/insights). GraphOS Studio lets you analyze data from failed requests, such as GraphQL error messages ([if enabled](/graphos/routing/graphos-reporting#errors)) and the ID of the client making the request. You can also [segment your insights data](/graphos/platform/insights/client-segmentation) based on the client ID.
+
+<Tip>
+
+[Learn how to ensure client IDs are included in all requests.](/graphos/routing/observability/client-id-enforcement)
+
+</Tip>
+
+## Enabling additional telemetry
+
+Use the methods below if you need information outside of what's presented in GraphOS Studio to debug client requests.
```

Comment:
```suggestion
To get data outside of GraphOS Studio, you can configure the GraphOS Router to generate logs and export telemetry metrics about incoming HTTP client requests. Because the likely high volume of client requests can generate significant telemetry data and incur steep observability costs, the router supports [configuration](/graphos/routing/observability/telemetry) of the exact telemetry you want your router to collect and export. 
```